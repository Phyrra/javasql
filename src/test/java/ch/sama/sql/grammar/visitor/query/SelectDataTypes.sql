select convert(datetime, '2015-02-21', 21), cast(int, '1'), cast(float, '1.23'), convert(varchar, dt, 21), cast(varchar(64), '1.23')