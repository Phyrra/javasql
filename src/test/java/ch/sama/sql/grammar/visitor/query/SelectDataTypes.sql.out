SELECT fnc(int, float, datetime, varchar, varchar(64), varchar(MAX))